assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
As a good practice, it is better to keep the initial state constant, i.e. not depending on the window object and pass the initial state when you create it. The function `[createStore](http://redux.js.org/docs/api/createStore.html)` receives the initial state as a parameter. In addition, if the actions do not depend on the window object, it is easier for us to test them with Jest. See [here](https://github.com/guardian/support-frontend/blob/master/assets/pages/bundles-landing/bundlesLanding.jsx#L25) for an example of this.